File: Assignments/Assignment_4/program1.c
// Write a program which accept number from user and display its multiplication of factors.

#include<stdio.h>

////////////////////////////////////////////////////////////////////
//
//  Function Name : MultiFact
//  Description :   It is used to display multiplication of factors
//  Input :         Integer
//  Output :        Integer
//  Author :        Omkar Sachin Narlawar
//  Date :          26/10/2025
//
////////////////////////////////////////////////////////////////////

int MultFact(int iNo)
{
    int iCnt = 0;
    int iMulti = 1;

    // Updater
    if(iNo < 0)
    {
        iNo = -iNo;
    }

    for (iCnt = 1; iCnt <= (iNo / 2); iCnt++)
    {
        if((iNo % iCnt) == 0)
        {
            iMulti = iMulti * iCnt;
        }
    }
    return iMulti;
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter number\n");
    scanf("%d",&iValue);

    iRet = MultFact(iValue);

    printf("%d",iRet);

    return 0;
}

File: Assignments/Assignment_67/program1.java
// Write a program which accept one number from user and count number of ON(1) bits in it without using % and / operator.

import java.util.Scanner;

class Bitwise
{
    public int CountOne(int iNo)
    {
        int iCnt = 0;

        while(iNo != 0)
        {
            if((iNo & 1) == 1)
            {
                iCnt++;
            }
            iNo = iNo >> 1;
        }
        return iCnt;
    }
}

class program1
{
    public static void main(String A[])
    { 
        int iValue = 0;
        int iRet = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Number : ");
        iValue = sobj.nextInt();

        Bitwise bobj = new Bitwise();
        iRet = bobj.CountOne(iValue);

        System.out.println("Number of ON bits are : "+iRet);   
    }
}